flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java

```diff
@@ -153,6 +161,8 @@
 
 	protected transient Context context = new Context(null, null);
 
+	protected transient WindowContext windowContext = new WindowContext(null);
```

Comment:
I think to make it more clear what they do we should rename these two contexts to `triggerContext` and `processContext`.